process/consensus.md

```diff
@@ -23,11 +23,12 @@ Polling consensus is done by the chair:
 2. The chair asks all participants to express their opinion to the question,
    asking in turn whether they are `Strongly For`, `For`, `Neutral`, `Against`,
    or `Strongly Against`. Participants vote for a single option by raising their
-   hand, or abstain entirely. Aggregate votes are recorded by the note-taker.
-3. If deemed relevant, the chair can ask certain participants if their wish to
+   hand, voting on a poll, or in chat for online meetings, or abstain entirely.
```

Comment:
How about this wording: "Participants either abstain entirely or vote for a single option by raising their hand if meeting in person or by voting on a virtual poll or commenting in chat if meeting online."

Reply:
or perhaps even "if attending in-person/online" since in practice we have both in a single meeting.